lib/core/test/unit/Cardano/Wallet/DB/Arbitrary.hs

```diff
@@ -463,26 +467,28 @@ rootKeysSeq = unsafePerformIO $ generate (vectorOf 10 genRootKeysSeq)
   where
     genRootKeysSeq :: Gen (ShelleyKey 'RootK XPrv)
     genRootKeysSeq = do
-        (s, g, e) <- (,,)
-            <$> genPassphrase @"seed" (16, 32)
-            <*> genPassphrase @"generation" (0, 16)
-            <*> genPassphrase @"encryption" (0, 16)
+        s <- (SomeMnemonic <$> genMnemonic @12)
+        g <- frequency
+                [ (1, return Nothing)
+                , (3, Just . SomeMnemonic <$> genMnemonic @12)
+                ]
+        e <- genPassphrase @"encryption" (0, 16)
         return $ Seq.generateKeyFromSeed (s, g) e
 {-# NOINLINE rootKeysSeq #-}
 
 arbitrarySeqAccount
     :: ShelleyKey 'AccountK XPub
 arbitrarySeqAccount =
-    publicKey $ unsafeGenerateKeyFromSeed (seed, mempty) mempty
+    publicKey $ unsafeGenerateKeyFromSeed (mw, Nothing) mempty
   where
-    seed = Passphrase $ BA.convert $ BS.replicate 32 0
+    mw = SomeMnemonic $ unsafePerformIO $ generate $ genMnemonic @15
 
 arbitraryRewardAccount
     :: ShelleyKey 'AddressK XPub
 arbitraryRewardAccount =
-    publicKey $ unsafeGenerateKeyFromSeed (seed, mempty) mempty
+    publicKey $ unsafeGenerateKeyFromSeed (mw, Nothing) mempty
   where
-    seed = Passphrase $ BA.convert $ BS.replicate 32 0
+    mw = SomeMnemonic $ unsafePerformIO $ generate $ genMnemonic @15
```

Comment:
Should use `unsafeMkSomeMnemonicFromEntropy` here now, to keep the old behaviour.